README.md

```diff
@@ -350,17 +350,24 @@ domain=127.0.0.1
 port=8443
 scheme=http
 secret=the_secret
```

Comment:
```suggestion
secret=secret
```
The default secret for Fauna Dev is `secret`. We don't make that a "secret": we show that in the last step of this procedure: https://docs.fauna.com/fauna/current/integrations/dev#run